killbill-integration-tests/core/test_account.rb

```diff
@@ -132,5 +132,32 @@ def test_account_update
 
     end
 
+    def test_account_missing_calls
+
+      account = create_account(@user, @options)
+
+      assert_respond_to(account, :blocking_states)
+      assert_respond_to(account, :block_account)
+      assert_respond_to(account, :cba_rebalancing)
+      assert_respond_to(account, :email_notifications)
+      assert_respond_to(account, :invoice_payments)
+
+      blocking_states = account.blocking_states('ACCOUNT','', 'NONE', @options).response.kind_of? Net::HTTPSuccess
+      requested_date = "2018-03-08"
+      block_account = account.block_account('test', 'test', false, false, false, requested_date, @user, nil, nil, @options)
+      cba_rebalancing = account.cba_rebalancing(@user, nil, nil, @options).response.kind_of? Net::HTTPSuccess
```

Comment:
This call would be **very** hard to verify _logically_, so don't bother too much about it: just have a separate method `test_cba_rebalancing` and keep a simple assertion like you have (`.response.kind_of? Net::HTTPSuccess`).